examples/getting-started-movielens/03-Training-with-TF.ipynb

```diff
@@ -2,7 +2,7 @@
  "cells": [
```

Comment:
<p>I think the paragraph doesn't make much sense</p><p> </p><p>Applying deep learning models to recommendation systems faces unique challenges in comparison to other domains, such as computer vision and natural language processing. The datasets and common model architectures have unique characteristics, which require custom solutions. Recommendation system datasets have terabytes in size with billion examples but each example is represented by only a few bytes. For example, the <a href="https://ailab.criteo.com/download-criteo-1tb-click-logs-dataset/" target="_blank">Criteo CTR dataset</a>, the largest publicly available dataset, is 1.3TB with 4 billion examples. The model architectures have normally large embedding tables for the users and items, which do not fit on a single GPU. </p><p> </p><p>We do not address large embeddings in the example - I dont know why we mention it.</p><p>We do not use Criteo dataset, not sure why we explain something related to it?</p><p> </p><p>CAn we rename NVTabular dataloader to Merlin dataloader?</p><p> </p><p><code>This notebook explains, how to use the NVTabular dataloader to accelerate TensorFlow training.</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NVIDIA-Merlin/Merlin/pull/779/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
<p>completely changed this paragraph now</p>